### PR TITLE
Fix line endings in Magic Leap scripts

### PR DESCRIPTION
**Pull request by chrisplatorres**
This PR proposes to fix line endings in Magic Leap scripts using `dos2unix` for the purpose of building in the Ubuntu WSL per the build instructions.

### CODE REVIEW

File: scripts/magicleap/version-ml.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
-
-export MLSDK=${MLSDK:-/mnt/c/Users/avaer/MagicLeap/mlsdk/v0.19.0}
-export MLSDK_WIN=$(echo "$MLSDK" | sed 's/^\/mnt\/\([a-z]\)\//\1:\\/' | sed 's/\//\\/g')
+#!/bin/bash
+
+export MLSDK=${MLSDK:-/mnt/c/Users/chris/MagicLeap/mlsdk/v0.19.0}
```
Comment (chrisplatorres) [inline scripts/magicleap/version-ml.sh]: unnecessary directory name change here

**Comment by modulesio** [comment]
Was this erroring otherwise?

**Comment by chrisplatorres** [comment]
Yes, dos2unix and `chmod +x` was required to run on the Ubuntu WSL.

File: deps/exokit-bindings/webglcontext/src/webgl.cc [hunk 2]
```diff
@@ -4148,7 +4148,7 @@ NAN_METHOD(WebGLRenderingContext::GetBufferSubData) {
     dataLength = std::min<size_t>(dataLength, length);
   }
 
-#if defined(ANDROID) && !defined(LUMIN)
+#if defined(ANDROID) || defined(LUMIN)
```
Comment (modulesio) [inline deps/exokit-bindings/webglcontext/src/webgl.cc]: Wrong PR?

Comment (chrisplatorres) [inline deps/exokit-bindings/webglcontext/src/webgl.cc]: carried over from https://github.com/exokitxr/exokit/pull/1384 , did not reset branch to master

File: scripts/magicleap/version-ml.sh (hunk 1 above)
Comment (chrisplatorres) [reply scripts/magicleap/version-ml.sh]: 921e7cb

**Comment by chrisplatorres** [comment]
@modulesio e7d0f7f

**Pull request merged**